modules/system/aoe.js

```diff
@@ -68,7 +68,7 @@ export default class AbilityTemplate extends MeasuredTemplate {
     let effect = fromUuidSync(effectUuid);
     // Sometimes, the radius needs to reference the test (usually overcasting)
     setProperty(effect, "flags.wfrp4e.sourceTest",  game.messages.get(messageId)?.getTest());
-    radius = radius || effect.radius; 
+    radius = (radius || effect.radius) / 2;
 
```

Comment:
This seems incorrect, as it's not checking for spells, also this has already been handled in https://github.com/moo-man/WFRP4e-FoundryVTT/commit/0af72d54b768699652accbac72aa4e8d5d9071ff,